src/Arcus.Templates.WebApi/Arcus.Templates.WebApi.csproj

```diff
@@ -56,7 +56,7 @@
     <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="2.2.0" />
-    <PackageReference Include="Arcus.WebApi.Logging" Version="0.2" />
+    <PackageReference Include="Arcus.WebApi.Logging" Version="0.3.0" />
     <PackageReference Include="Arcus.WebApi.Security.Authentication" Version="0.2" Condition="'$(Auth)' == 'true'" />
```

Comment:
This should also be updated ;) I think here's your problem with the build.

Reply:
Ah it looks like the Package Manager doesn't like those conditions 🙄 
![image](https://user-images.githubusercontent.com/4345663/69715882-4e0bde80-1109-11ea-9b6a-ed556adb6327.png)